src/main/java/codeu/model/data/Setting.java

```diff
@@ -1,16 +1,40 @@
 package codeu.model.data;
 
 public class Setting {
-    private String defaultTheme;
+    SettingType type;
+    String value;
 
-    public Setting(String  defaultTheme){
-        this.defaultTheme = defaultTheme;
+    public Setting(SettingType type, String value) {
```

Comment:
is there a way to set the value right now. Without creating a whole new setting. The only place using this constructor should be PersistentDataStore. Every other place should be using some sort of update method. PersistentDataStore.java is the only place actually creating the object. The rest of our system should only have the option of using the object already created and updated it.

Reply:
Hmm! Okay. I will try to set it right now.